backend/src/main/java/woowacourse/touroot/authentication/infrastructure/JwtTokenProvider.java

```diff
@@ -0,0 +1,36 @@
+package woowacourse.touroot.authentication.infrastructure;
+
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import java.util.Date;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import woowacourse.touroot.member.domain.Member;
+
+@Component
+public class JwtTokenProvider {
+
+    private static final String MEMBER_ID_KEY = "id";
+
+    private final String secretKey;
+    private final long validityInMilliseconds;
+
+    public JwtTokenProvider(
```

Comment:
```suggestion
    public JwtTokenProvider(
            @Value("${security.jwt.token.secret-key}") String secretKey,
            @Value("${security.jwt.token.expire-length}") long validityInMilliseconds
    ) {
        this.secretKey = secretKey;
        this.validityInMilliseconds = validityInMilliseconds;
    }
```
괄호 내려주면 다른 파일이랑 컨벤션이 더 잘 맞을 것 같습니다

Reply:
반영 완!